ebos/eclwriter.hh

```diff
@@ -396,7 +396,7 @@ public:
 
             // then, make sure that the previous I/O request has been completed and the
             // number of incomplete tasklets does not increase between time steps
-            taskletRunner_->barrier();
+            // taskletRunner_->barrier();
```

Comment:
@goncalvesmachadoc: Could you comment on why this barrier was introduced?

Reply:
> [Why was] this barrier was introduced?

Just to avoid confusion: This line has been in place since commit 8f61fe03, and was originally introduced in opm-modules as commit OPM/opm-models@0fb23919 (PR OPM/opm-models#299).

Reply:
I re-enabled the barrier to avoid problems I might not be aware of.